kolibri/utils/server.py

```diff
@@ -8,9 +8,11 @@
 import requests
 from django.conf import settings
 from django.core.management import call_command
-from kolibri.content.utils import paths
 
-from .system import kill_pid, pid_exists
+from .system import kill_pid
+from .system import pid_exists
+from kolibri.content.utils import paths
```

Comment:
This should not actually be here, as it has a potential to break module dependency. From `kolibri.utils.__init__`:

```
"""
Do not import anything from the rest of Kolibri in this module, it's crucial
that it can be loaded without the settings/configuration/django stack!
"""
```

Reply:
Perhaps we could pull it out into utils, and reverse the import the other way?

Reply:
I can't quite grasp how this module design should be... it's already pretty complicated :) Maybe we should open an issue? Trying to think of a framing.. maybe the issue is to figure out how to make `kolibri.utils` independent from other modules for now?